src/components/NFTCollection/CollectionItem.css

```diff
@@ -0,0 +1,58 @@
+.collection-item {
+    display: flex;
+    height: 50%;
+    flex-direction: row;
+    flex: 3;
+    justify-content: space-between;
+    align-items: left;
+    padding: 0.5rem;
+    margin: 1rem 0;
+    background-color: #ba5a5a;
+  }
+  
+  .collection-item__description {
+    display: flex;
+    height: 60%;
+    flex-direction: row;
+    gap: 1rem;
+    align-items: flex-end;
+    flex-flow: column-reverse;
+    justify-content: flex-start;
+    flex: 5;
+  }
+  
+  .collection-item h2 {
+    color: #3a3a3a;
+    font-size: 1rem;
+    flex: 1;
+    margin: 0 1rem;
+    color: white;
+  }
```

Comment:
This is one way to style a `h2` tag, but do you think that it might be neater to give the tag a class and style through a class name?